.eslintrc.js

```diff
@@ -20,4 +24,10 @@ module.exports = {
       '@metamask/eslint-config/config/nodejs',
     ],
   }],
-};
\ No newline at end of file
+
+  ignorePatterns: [
+    '!.eslintrc.js',
+    'dist/',
+    'node_modules/',
```

Comment:
This entry is not necessary - [`node_modules` is always ignored](https://eslint.org/docs/user-guide/configuring/ignoring-code), along with all dotfiles. That's why the `!.eslintrc.js` entry is required.